content/reference/redactors/overview.md

```diff
@@ -0,0 +1,73 @@
+---
+date: 2019-10-23
+linktitle: "Redactors Overview"
+title: "Redactors Overview"
+weight: 20010
+---
+
+Redactors are YAML specifications that define which data to remove when generating a support bundle.
+
+Redactors can be specified as a standalone document, as a global entry in a collector spec, or individually on a single collector.
+
+```yaml
+apiVersion: troubleshoot.replicated.com/v1beta1
+kind: Redactor
+metadata:
+  name: my-redactor-name
+spec:
+  redacts: []
+```
+
+```yaml
+apiVersion: troubleshoot.replicated.com/v1beta1
+kind: Collector
+metadata:
+  name: my-collector-name
+spec:
+  collectors:
+  - data:
+      redactors: []
+  globalRedactors: []
+```
+
+Each redactor consists of a set of files which it can apply to, a set of string literals to replace, and a set of regex replacements to be run.
```

Comment:
This should be rewritten into paragraph form, instead of 1 sentence per line.

Reply:
Markdown turns this into a single paragraph already - one sentence per line is to make git diffs a bit smaller